### DISCUSSION TITLE
Docker Initial Setup

### DISCUSSION THREAD

**Discussion by ajb3932** [dfirtrack/dfirtrack#85] [Q&A]
![Capture](https://user-images.githubusercontent.com/45558436/114845758-ede80c00-9dd3-11eb-9171-30d7273c2ccf.JPG)
Hi,

Maybe its a noob question but I have ran docker compose and when I try to access the server I just get a 400 Bad Request. How do I access the platform once it is running? 

Many Thanks!

Alex

**Comment by stuhli** [comment]
Most likely you forgot to change `fqdn` to `localhost` within `docker/.env`? The default value is `dfirtrack.test` which generates an error if you are trying to access Django via localhost. You may have to run `docker-compose up --build` afterward to rebuild.

If this does not solve the problem, please change `DEBUG` to `True` within `docker/local_settings.py` and run `docker-compose up --build`. This should give us more information about the problem.

**Reply by ajb3932** [reply]
Changing fqdn worked a treat! Thank you. Next item is the default Username and Password. Did I miss somewhere to add, change or view it? Sorry!

**Reply by stuhli** [reply]
Yeah, after running `docker-compose up` the last hint on the commandline is "You may run docker/setup_admin.sh from the host system to create a new superuser". I admit, it's a bit hidden. I will add this to the readme when I get a chance.

For now open a commandline (while running the container in another commandline or demon mode) and execute `docker/setup_admin.py`.

**Reply by ajb3932** [reply]
![image](https://user-images.githubusercontent.com/45558436/114861417-26dcac80-9de5-11eb-955a-d6c20228b186.png)

Thank you again! I updated the username and password and logged in and receive a 500 response. Have I done anything wrong? Hopefully this is the last thing!

**Reply by stuhli** [reply]
So, the login screen appeared? In this case please set `DEBUG=True` like described above. The error message is too generic and I have no clue what might be the remaining problem.
